galasa-managers-parent/galasa-managers-common-parent/dev.galasa.common/src/main/java/dev/galasa/SSLTLSContextName.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package dev.galasa.common;
+
+import java.util.Properties;
+
+public class SSLTLSContextName {
+
+	public static final String JAVA_VENDOR_PROPERTY = "java.vendor";
+    public static final String JAVA_VERSION_PROPERTY = "java.version";
+    
+    
+    public static String getSelectedSSLContextName( Properties... props ) {
```

Comment:
Im curious why you are saying that you need a variable number of properties containers here ?
Seems that you always need one ?
System.getProperties is a pig to unit test, so it's better if you always pass the properties into this method ?

Also, I would personally like it if this wasn't a static method.
The classes that use it then have a dependency which can't easily be mocked out.